debian/control

```diff
@@ -38,6 +38,7 @@ Depends: cloud-guest-utils | cloud-utils,
          netplan.io,
          procps,
          python3,
+         python3-debconf,
```

Comment:
We also need this in Build-Depends too. See https://github.com/canonical/cloud-init/pull/1511 for context.